jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -417,6 +417,36 @@ public void sendMessage(
         }
     }
 
+    /**
+     * Sends a message that originated from a relay, forwarding it to local endpoints
+     * and to those relays with different mesh-id values than the source relay
+     *
+     * @param msg the message to be sent
+     * @param meshId the ID of the mesh from which the message was received.
+     */
+    public void sendMessageFromRelay(
+        BridgeChannelMessage msg,
+        @Nullable String meshId)
+    {
+        for (Endpoint endpoint : getLocalEndpoints())
+        {
+            endpoint.sendMessage(msg);
+        }
+
+        for (Relay relay: relaysById.values())
+        {
+            if (!Objects.equals(meshId, relay.getMeshId()))
+            {
+                relay.sendMessage(msg);
+            }
+        }
+
+        if (tentacle != null && meshId != null)
```

Comment:
This condition is `true` only if we use old octo with mesh IDs?

Reply:
Yeah, which is impossible, but I was trying to be complete...do you think I should just have a comment saying it cannot happen for old octo?